packages/node_modules/@webex/plugin-meetings/test/unit/spec/member/util.js

```diff
@@ -0,0 +1,45 @@
+import {assert} from '@webex/test-helper-chai';
+
+import MemberUtil from '@webex/plugin-meetings/src/member/util';
+
+
+describe('isHandRaised', () => {
+  it('throws eroor when there is no participant', () => {
+    assert.throws(() => {
+      // eslint-disable-next-line no-unused-vars
+      MemberUtil.isHandRaised();
+    }, 'Raise hand could not be processed, participant is undefined.');
+  });
+
+  it('return false when hand is not there in controls', () => {
+    const participant = {
+      controls: {}
+    };
+
+    assert.deepEqual(MemberUtil.isHandRaised(participant), false);
```

Comment:
no need for deepEqual, a normal "Equal" is also fine - applies for other occurrences as well.
If you are not familiar, pls read up the difference between deepEqual and equal